public/angular/js/controllers/application.js

```diff
@@ -47,6 +48,14 @@ angular.module('Aggie')
       g.src='https://cdn.matomo.cloud/' + data.matomo.dashboard_name +'.matomo.cloud/container_' + data.matomo.container_id  + '.js'; 
       s.parentNode.insertBefore(g,s);
 
+      // Sending Username Data
+      $rootScope.$watch('currentUser', function() {
+        if ($rootScope.currentUser != undefined) {
+          var user = $rootScope.currentUser;
+          _paq.push(["setUserId", user.username]);
+        }      
```

Comment:
Have you tested this with different usernames -- what happens if you sign out and sign back in as someone else? Just want to make sure it properly attributes in that case.

Let's also add an `else` condition to push `["setUserId", null]`, that seems simple enough right?